### PR TITLE
added getObjectValue helper method

### PR DESCRIPTION
**Pull request by hrestakd**
TreeGrid and QuickGrid components now support getting cell values from nested objects.

For this scenario the proper format type of ValueMember or DataMember is `object.property`

### CODE REVIEW

File: src/components/TreeGrid/TreeGrid.tsx [hunk 1]
```diff
@@ -266,30 +267,6 @@ export class TreeGrid extends React.PureComponent<ITreeGridProps, ITreeGridState
         );
     }
 
-    private formatBoolCell(cellData: any, rowData: any) {
```
Comment (vinkobedek) [inline src/components/TreeGrid/TreeGrid.tsx]: what happened here? leftover from a previous move?

File: src/utilities/getObjectValue.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+ /**
+ * Get value for object from path.
+ * @param {string} path - Property name of object. For nested objects use property.nested
+ */
+export function getObjectValue(theObject: any, path: string, separator = '.') {
```
Comment (vinkobedek) [inline src/utilities/getObjectValue.ts]: how's the performance of this function? try it with 400 000 elements vs 400 000 elements in the old version just to be on the safe side.

File: src/components/TreeGrid/TreeGrid.tsx (hunk 1 above)
Comment (hrestakd) [reply src/components/TreeGrid/TreeGrid.tsx]: Yes, it's a sneaky delete, I forgot to clear this last time :)

File: src/utilities/getObjectValue.ts (hunk 2 above)
Comment (hrestakd) [reply src/utilities/getObjectValue.ts]: I've done some testing, both components on 400-500k rows work almost equally fast

**Pull request merged**